tgui/packages/tgui/interfaces/TeleConsole/index.tsx

```diff
@@ -28,41 +35,139 @@ export const TeleConsole = (_props, context) => {
+                  onRestoreBookmark={handleRestoreBookmark}
+                  targetCoords={[xTarget, yTarget, zTarget]}
+                />
+                <ConnectionSection
+                  isConnected={isConnectedToHost}
+                  isPanelOpen={!!isPanelOpen}
+                  onCyclePad={handleCyclePad}
+                  onReset={handleResetConnect}
+                  onRetry={handleRetryConnect}
+                  padNum={padNum}
+                />
+              </Section>
+            </Stack.Item>
+          )}
+          {(tab===Tab.LongRange) && (
+            <Stack.Item>
+              <Section>
+                <Section title="Destinations">
+                  <LabeledList>
+                    {destinations.length ? destinations.map((d) => (
+                      <div key={d["name"]}>
+                        <LabeledList.Item
+                          label={d["name"]}>
+                          <Box textAlign="right">
+                            <Button
+                              icon="sign-out-alt"
+                              onClick={() => act("lrt_send", { name: d["name"] })}
+                              disabled={!isConnectedToHost}
+                            >
+                              Send
+                            </Button>
+                            <Button
+                              icon="sign-in-alt"
+                              onClick={() => act("lrt_receive", { name: d["name"] })}
+                              disabled={!isConnectedToHost}
+                            >
+                              Receive
+                            </Button>
+                            <Button onClick={() => act('lrt_portal', { name: d["name"] })} disabled={!isConnectedToHost}>
+                              <Icon name="ring" rotation={90} />
+                              Toggle Portal
+                            </Button>
+                          </Box>
+                        </LabeledList.Item>
+                      </div>
+                    )) : (
+                      <LabeledList.Item>
+                        No destinations are currently available.
+                      </LabeledList.Item>
+                    )}
+                  </LabeledList>
+                </Section>
+                {readout && <Section>{formatReadout(readout)}</Section>}
+                <ConnectionSection
+                  isConnected={isConnectedToHost}
+                  isPanelOpen={!!isPanelOpen}
+                  onCyclePad={handleCyclePad}
+                  onReset={handleResetConnect}
+                  onRetry={handleRetryConnect}
+                  padNum={padNum}
+                />
+              </Section>
+              {(!!disk) && (
+                <Section
+                  title="Disk Controls"
+                >
+
+                  <Button
+                    icon="upload"
+                    color={"blue"}
```

Comment:
Color override needed?

Reply:
Results in a different tint of blue.

Reply:
Right, but is that a problem? Consistent color scheme vs. aesthetic preference is a rough discussion, sure.

Reply:
It is if that is the desired behavior.  Removed in favor of consistent color scheme.